src/routes/+page.svelte

```diff
@@ -548,6 +550,106 @@
 		</div>
 	</div>
 </section>
+<section>
+	<div class="row">
```

Comment:
Missing section id and title:

```suggestion
<section id="menu">
        <Typography variant="h1">Menu</Typography>
	<div class="row">
```